crates/matrix-sdk/src/http_client.rs

```diff
@@ -161,7 +161,7 @@ impl HttpClient {
 
         let request = request.map(|body| body.freeze());
 
-        trace!("Sending request");
+        trace!("Sending request: {:?} with config {:?}", request, config);
```

Comment:
I think we should use structured logging for the metadata, i.e. include them as fields rather than part of the message.